frontend/src/components/filters/punishments/PunishmentTagSelect.svelte

```diff
@@ -0,0 +1,106 @@
+<script lang="ts">
+  import type { PunishmentType } from "src/types";
+  import { group } from "../../../stores/groups";
+  import Svelecte, { addFormatter, config } from "svelecte";
+  import { punishmentsToFilter } from "../../../stores/punishmentToFilter";
+
+  const myI18n = {
+    empty: `Alle straffer vises`,
+    nomatch: "Ingen matchende straffer",
+  };
+
+  let value: string;
+
+  const addPunishment = async (punishmentInput: PunishmentType) => {
+    $punishmentsToFilter = [
+      ...$punishmentsToFilter,
+      {
+        name: punishmentInput.name,
+        logo_url: punishmentInput.logo_url,
+        value: punishmentInput.value,
+        punishment_type_id: punishmentInput.punishment_type_id,
+      },
+    ];
+  };
+
+  function colorRenderer(punishment: PunishmentType, isSelected: boolean) {
+    if (isSelected) {
+      addPunishment(punishment);
+      value = null;
+    }
+    return `<div class="flex flex-row w-fit"><img class="px-0.5" src=${punishment.logo_url}></img><p>${punishment.name}</p></div>
+     `;
+  }
+
+  addFormatter({
+    "pun-blocks": colorRenderer,
+  });
+</script>
+
+<div class="flex flex-col w-full mt-2">
+  <label for="punishment"
+    ><span class="label-text mb-2">Vis strafftyper</span></label
+  >
+  <Svelecte
+    options="{$group.punishment_types.filter(
+      (filterPun) =>
+        !$punishmentsToFilter
+          .map((groupPun) => groupPun.punishment_type_id)
+          .includes(filterPun.punishment_type_id)
+    )}"
+    i18n="{myI18n}"
+    renderer="pun-blocks"
+    inputId="punishment"
+    bind:value
+    placeholder="Navn på straff"
+  />
+</div>
+
+<style lang="less">
```

Comment:
```suggestion
<style lang="postcss">
```

just thinking since we're not really using less anyway. I think it might be better to use the color values we have in TW for consistency as well